qa-tester/assets/src/js/admin-bar/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Internal dependencies
+ */
+import Container from './components/container';
+
+domReady( () => {
+	render(
+		<Container />,
+		document.getElementById( 'amp-qa-tester-build-selector' )
```

Comment:
Should check that this element exists first because there might be cases where the admin bar doesn't show on pages that are technically admin pages (such as the setup wizard)

Reply:
Good catch :+1:.